Common/NPCs/GlobalLoot.cs

```diff
@@ -14,5 +15,7 @@ public override void ModifyGlobalLoot(Terraria.ModLoader.GlobalLoot globalLoot)
 		globalLoot.Add(ItemDropRule.ByCondition(new BossDownedCondition(BossDownedCondition.Bosses.Deerclops), ModContent.ItemType<DeerclopsMap>(), 1000));
 		globalLoot.Add(ItemDropRule.ByCondition(new BossDownedCondition(BossDownedCondition.Bosses.Skeletron), ModContent.ItemType<SkeletronMap>(), 1000));
 		globalLoot.Add(ItemDropRule.ByCondition(new BossDownedCondition(BossDownedCondition.Bosses.WallOfFlesh), ModContent.ItemType<WoFMap>(), 1000));
+
+		globalLoot.Add(ItemDropRule.ByCondition(new BossDownedCondition(BossDownedCondition.Bosses.Skeletron), ModContent.ItemType<AugmentationOrb>(), 500));
```

Comment:
Am just heading to bed, so can't give this a proper review but we'll want to add this as apart of the drop table system and not have it just be a generic 1/500
I'm thinking it could be apart of the Currency drop pool